docs/source/deploying-cloud.rst

```diff
@@ -0,0 +1,75 @@
+Cloud Deployments
```

Comment:
I think this should maybe just be called "Cloud" given that it's already under the "Deploying Clusters" section